manager/integration/tests/backupstore.py

```diff
@@ -170,6 +170,48 @@ def nfs_get_backup_blocks_dir():
     raise NotImplementedError
 
 
+def backupstore_create_file(client, core_api, prefix, file_name, data={}):
+    backup_target_setting = client.by_id_setting(SETTING_BACKUP_TARGET)
+    backupstore = backup_target_setting.value
+
+    if is_backupTarget_s3(backupstore):
+        return mino_create_file_in_backupstore(client,
+                                               core_api,
+                                               prefix,
+                                               file_name,
+                                               data)
+    else:
+        raise NotImplementedError
+
+
+def mino_create_file_in_backupstore(client, core_api, prefix, file_name, data={}): # NOQA
+    backup_target_credential_setting = client.by_id_setting(
+        SETTING_BACKUP_TARGET_CREDENTIAL_SECRET)
+
+    secret_name = backup_target_credential_setting.value
+
+    minio_api = minio_get_api_client(client, core_api, secret_name)
+    bucket_name = minio_get_backupstore_bucket_name(client)
+
+    if len(data) == 0:
+        data = {"testkey": "test data from mino_create_file_in_backupstore()"}
+
+    temp_file_path = "/tmp/temp_file.cfg"
```

Comment:
It's better not to hard code `temp_file_path` in the util function.

Reply:
fixed